source/io/module_httpClient.js

```diff
@@ -71,45 +70,7 @@
         // None
 
         // Static Public Functions
-        // https://fetch.spec.whatwg.org/#forbidden-header-name
-        var forbiddenHeaderNames = [
```

Comment:
Since we have spent time finding the forbidden header names, I would leave a comment with an explanation and all these names.